// ngx-scanner/fesm5/scanner.js
import { __decorate, __metadata } from 'tslib';
import { Component, NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';

var ScannerIneComponent = /** @class */ (function () {
    function ScannerIneComponent() {
    }
    ScannerIneComponent.prototype.ngOnInit = function () {
    };
    ScannerIneComponent = __decorate([
        Component({
            selector: 'app-scanner-ine',
            template: "<h1>\n  <ng-content></ng-content>\n</h1>\n",
            styles: ["h1{color:red}"]
        }),
        __metadata("design:paramtypes", [])
    ], ScannerIneComponent);
    return ScannerIneComponent;
}());

var ScannerIneModule = /** @class */ (function () {
    function ScannerIneModule() {
    }
    ScannerIneModule = __decorate([
        NgModule({
            declarations: [
                ScannerIneComponent
            ],
            imports: [
                CommonModule
            ],
            exports: [
                ScannerIneComponent // <-- this!
            ]
        })
    ], ScannerIneModule);
    return ScannerIneModule;
}());

/**
 * Generated bundle index. Do not edit.
 */

export { ScannerIneModule, ScannerIneComponent as ɵa };
